ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -0,0 +1,108 @@
+package org.openmrs.reference;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.reference.page.HeaderPage;
+import org.openmrs.reference.page.HtmlFormsPage;
+import org.openmrs.reference.page.ManageHtmlFormsPage;
+import org.openmrs.reference.page.HomePage;
+import org.openmrs.reference.page.ManageFormsPage;
+import org.openqa.selenium.By;
+import org.openmrs.uitestframework.test.TestBase;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by nata on 24.06.15.
```

Comment:
Git tracks the java class `[FormTest]` to be newly created and am wondering whether `nata` is the java class author !

Reply:
Any response @sherrif10 on the comment above